modules/jolt_physics/spaces/jolt_query_collectors.h

```diff
@@ -189,9 +190,10 @@ template <typename TBase, int TDefaultCapacity>
 class JoltQueryCollectorClosestMulti final : public TBase {
 public:
 	typedef typename TBase::ResultType Hit;
+	typedef JPH::Array<Hit, JPH::STLLocalAllocator<Hit, TDefaultCapacity + 1>> HitArray;
```

Comment:
One thing I just realized: Because `TDefaultCapacity + 1` is not a power of 2 and because the `Array` grows in powers of two, you will actually overflow to heap when you add the `TDefaultCapacity + 1`-th element. To get around this you should call `reserve(TDefaultCapacity + 1)` in the constructor of `JoltQueryCollectorClosestMulti`. I'll change the `Array` class so that it is aware of `STLLocalAllocator` and will try to stay within the buffer. Please add the `reserve` here for now as it's a less risky change.

Reply:
Good catch! I had this fix in the extension as well, but forgot about it.